### PR TITLE
Add re-login information after successful action

### PR DESCRIPTION
**Pull request by sharidas**
After the command is executed successfully by
recreating masterkey, the user should be informed
to relogin.

Signed-off-by: Sujith H <sharidasan@owncloud.com>

### CODE REVIEW

File: lib/Command/RecreateMasterKey.php [hunk 1]
```diff
@@ -231,6 +231,7 @@ protected function execute(InputInterface $input, OutputInterface $output) {
 				$this->keyManager->validateMasterKey();
 				$this->encryptAllUsers($input, $output);
 				$output->writeln("\nEncryption completed successfully\n");
+				$output->writeln("\nNOTE: Kindly log-out and log-in to access the files encrypted with new master key.");
```
Comment (PVince81) [inline lib/Command/RecreateMasterKey.php]: "All users will need to log out and log in again".

Not only the admin...

File: tests/Command/RecreateMasterKeyTest.php [hunk 2]
```diff
@@ -197,6 +197,7 @@ function($path) {
 				->willReturn(true);
 
 			global $outputText;
+			global $reloginText;
```
Comment (PVince81) [inline tests/Command/RecreateMasterKeyTest.php]: don't use globals but `use (&$reloginText)` instead in the closure

Comment (sharidas) [reply tests/Command/RecreateMasterKeyTest.php]: Done

File: lib/Command/RecreateMasterKey.php (hunk 1 above)
Comment (sharidas) [reply lib/Command/RecreateMasterKey.php]: Done

**Comment by sharidas** [comment]
The imho CI failure doesn't look related to this change...

**Comment by PVince81** [comment]
@sharidas please backport

**Comment by sharidas** [comment]
Backport to stable10 : https://github.com/owncloud/core/pull/34596

**Pull request merged**